shell/files/activate.tmpl.sh

```diff
@@ -5,7 +5,7 @@ export HERMIT_ENV={{.Root}}
 if [ -n "${ACTIVE_HERMIT+_}" ]; then
   if [ "$ACTIVE_HERMIT" = "$HERMIT_ENV" ]; then
     echo "This Hermit environment has already been activated. Skipping" >&2
-    return 34
+    return 0
```

Comment:
While I'm in favor of this change, it would be good to somehow "reserve" the exit code 34 so that it's not used for any other error in the future. We have existing code in a number of places explicitly checking for this exit code to ignore it, and it would be bad if this exit code started being used for some other error in the future

Reply:
I've added some [basic linting](https://github.com/cashapp/hermit/pull/474/files#diff-f6b20c425ee57cd5dcc96e5b486a83154b7de95233b2e0bd9b0ff26e4be6f6e1) for it across the shell scripts. It's not perfect, but should hopefully help a bit.